.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: testsuite
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy: { matrix: { node_version: [10.x] } }
```

Comment:
Is it worth it to have the version Matrix? We're not a library, so we only really need to support one version of Node.js. Might also make the file more legible to just hardcode the node version into that `node-version` field.

Reply:
Yep good point. Will remove.

Though we are using a version of node that is now no longer even in maintenance mode as of May, seems worth a separate effort to upgrade to 16 or 17 (the current LTS). Will consider that separately.

Reply:
Sounds good. BTW, I did a very lightweight test of upgrading to Node 16, and it looks like we'll probably need to bump the version of node-sass we're using, since it's so old that node-sass doesn't have a precompiled version of itself for Node 16.